packages/taikoon-ui/src/routes/+layout.ts

```diff
@@ -2,4 +2,4 @@
 // which essentially means you turn your app into an SPA.
 // See https://kit.svelte.dev/docs/page-options#ssr
 
-export const ssr = false;
+//export const ssr = false;
```

Comment:
```suggestion
export const ssr = true;
```

Reply:
We need SSR enabled for the geolocation headers from Vercel

Reply:
And `ssr = true` does that